README.md

```diff
@@ -19,6 +19,37 @@ used to run the batch on your behalf. There's a dedicated Batch service
 account for each project (e.g. "tob-wgs"), which helps with bucket permission
 management and billing budgets.
 
+## Access through CLI
+
+If you have the `analysis-runner` CLI installed, you can request the analysis-runner
+starts jobs from a GitHub repository + commit.
+
+If you're in the directory of the project you want to run, you can omit
+the `--commit` and `--repository` parameters, which will use your current REMOTE
+and commit HEAD.
+
+For example:
+
+```bash
+cpg-analysisrunner \
```

Comment:
Can we call this analysis-runner, for consistency with the repository name?

Reply:
Renamed to `analysis-runner`.